.github/workflows/releaser.yml

```
@@ -119,9 +119,9 @@ jobs:
           cp ${CP_ARGS} artifacts/macos-dmg-X64-electron/Parsec_${{ needs.version.outputs.full }}_mac_*.unsigned{.dmg,.zip}{,.blockmap} release-files
           cp ${CP_ARGS} artifacts/macos-dmg-X64-electron/latest-mac-*.yml release-files
 
-          # EXE
-          cp ${CP_ARGS} artifacts/windows-exe-X64-electron/Parsec_${{ needs.version.outputs.full }}_win_*.unsigned.exe{,.blockmap} release-files
-          cp ${CP_ARGS} artifacts/windows-exe-X64-electron/latest-win-*.yml release-files
+          # # EXE
```

I'll specify the reason in a comment
```suggestion
          # Currently, Windows electron client is not signed by the CI, thus 
          # it should not be uploaded when creating the draft release.
          # These artifacts are uploaded manually by someone who is able to sign them. 
          # # EXE
```